src/ert/gui/tools/manage_experiments/storage_info_widget.py

```diff
@@ -230,6 +279,9 @@ def _currentItemChanged(
             elif "index" in observation_ds.coords:
                 observation_ds = observation_ds.sel(index=int(observation_label))
 
+            if scaling_ds:
+                _render_scaled_obs()
+
```

Comment:
Why do you need to do this twice?

Reply:
They appear within each their branch of the `if bool(response_ds.dims)` which checks if there are responses, so it will only ever be executed at most once. In both cases we can render the scaled observations. It seems though, like it is not possible to ever get into the state of having a scaled observation for a response `X`, but not the response `X` itself. I tried this for (1) not outputting the `GEN_DATA` file, and (2) outputting an empty `GEN_DATA` file, and it caused errors. Will remove one render.